code/game/objects/structures/aliens.dm

```diff
@@ -281,9 +281,9 @@
 							child.Attach(M)
 							break
 
-/obj/structure/alien/egg/Move()
-	remove_from_proximity_list(src, 1)
-	..()
+/obj/structure/alien/egg/Moved(oldloc)
+	remove_from_proximity_list(oldloc, 1)
```

Comment:
shouldn't these also add to proximity list in the new loc, or is there something else handling that?